dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Kozak.xml

```diff
@@ -93,112 +93,22 @@
-            <column name="tag_id">7</column>
-            <column name="language_id">1</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">20</column>
-            <column name="name">Green thinking</column>
-            <column name="tag_id">7</column>
-            <column name="language_id">2</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">21</column>
-            <column name="name">Зеленое мышление</column>
-            <column name="tag_id">7</column>
-            <column name="language_id">3</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">22</column>
-            <column name="name">Нульові відходи</column>
-            <column name="tag_id">8</column>
-            <column name="language_id">1</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">23</column>
-            <column name="name">Zero waste</column>
-            <column name="tag_id">8</column>
-            <column name="language_id">2</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">24</column>
-            <column name="name">Нулевые отходи</column>
-            <column name="tag_id">8</column>
-            <column name="language_id">3</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">25</column>
-            <column name="name">3 р'с</column>
-            <column name="tag_id">9</column>
-            <column name="language_id">1</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">26</column>
-            <column name="name">3 r's</column>
-            <column name="tag_id">9</column>
-            <column name="language_id">2</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">27</column>
-            <column name="name">3 р'с</column>
-            <column name="tag_id">9</column>
-            <column name="language_id">3</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">28</column>
-            <column name="name">Еко-місто</column>
-            <column name="tag_id">10</column>
-            <column name="language_id">1</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">29</column>
-            <column name="name">Eco-city</column>
-            <column name="tag_id">10</column>
-            <column name="language_id">2</column>
-        </insert>
-        <insert tableName="tag_translations">
-            <column name="id">30</column>
-            <column name="name">Эко-город</column>
-            <column name="tag_id">10</column>
-            <column name="language_id">3</column>
-        </insert>
```

Comment:
Don't remove the change set, just write new, that will delete the table or column that you need to delete

Reply:
Done